tools/prepare_private_id_set_for_merge.py

```diff
@@ -0,0 +1,65 @@
+import json
```

Comment:
is this a copy of an existing script?

Reply:
There is another script that is similar to it - but they are different in many ways